### DISCUSSION TITLE
Feature Request: dax query code window -> select code snippet -> rightclick context menu -> Run Selected Code

### DISCUSSION THREAD

**Discussion by seedjay1** [TabularEditor/TabularEditor3#1082] [Ideas]
First world problems, but still would be nice to NOT have to run myself up to the Query menu to do this. :)

Presumably this makes sense for other types of code windows too.

**Comment by otykier** [comment]
Could you be a little more specific? There is no "Run Selected Code" option in the right-click context menu of a DAX query editor in TE3 (instead, you just use the "Execute selection" toolbar button, which has the SHIFT+F5 shortcut).

**Comment by seedjay1** [comment]
Right - i would LIKE a context menu selection for that. I'm not so much a keyboard shortcut guy.

Totally possible it's just too small/niche an issue, but thought I'd mention it.

![image](https://user-images.githubusercontent.com/31414209/148559927-72587d2c-b292-4c04-92d9-2ba2565cc6d5.png)
